projects/lf-documentation/src/app/example-usage-in-angular/example-usage-in-angular.component.html

```diff
@@ -6,7 +6,10 @@ <h3 id="cdn-example-angular"> Example: <code>lf-checklist</code></h3>
 <ol>
   <li>
     <p  class="documentation-text">
-      Add the following script tags to <code>index.html</code>
+      Add the following script tag to <code>index.html</code>. </p>
+    <p  class="documentation-text"> Insert this script tag after <code>&lt;body&gt;</code>.
+
+      This script requires <code>Zone.js</code> to load before it loads, and Angular by default loads <code>Zone.js</code> in <code>polyfills.js</code> inside <code>&lt;body&gt;</code>.
```

Comment:
Maybe add another bullet that says, "If for some reason you do not already import zone.js in polyfills.ts you can add the script tag:"...
Not sure that's necessary though

Reply:
the script tag to add zone is missing?

Reply:
nice catch